shop/src/main/java/ro/msg/learning/shop/dto/ProductAndCategoryDto.java

```diff
@@ -0,0 +1,26 @@
+package ro.msg.learning.shop.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.UUID;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class ProductAndCategoryDto {
+    private UUID productId;
```

Comment:
are the id's needed in the dtos?

Reply:
I considered I need to be able to see the Id in the endpoint.

Reply:
ok